### ISSUE TITLE
The plugin `flutter_ffmpeg` uses a deprecated version of the Android embedding. 

### ISSUE THREAD

**Issue by Sulisong** [tanersener/flutter-ffmpeg#320]
I use the latest version of Flutter:
```
Flutter 2.4.0-5.0.pre.127 • channel master • https://github.com/flutter/flutter.git
Framework • revision 0a8c03dc8a (22 minutes ago) • 2021-07-21 12:51:43 -0700
Engine • revision 224fa96049
Tools • Dart 2.14.0 (build 2.14.0-339.0.dev)
```
vscode prompt:
```
The plugin `flutter_ffmpeg` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
```
![image](https://user-images.githubusercontent.com/40148554/126555513-a7a9afbd-5bd6-44e4-aa1c-c3aed47bdc21.png)

**Comment by Sulisong** [comment]
@tanersener

**Comment by tanersener** [comment]
Thanks for the PR. Changes are released in `v0.4.2`.

**Comment by Sulisong** [comment]
> Thanks for the PR. Changes are released in `v0.4.2`.

Hey-hey
